server/ngwmn/tests/services/test_ngwmn.py

```diff
@@ -6,10 +6,169 @@
 
 import requests as r
 import requests_mock
+import copy
+import ngwmn.services.ngwmn as ngwmn
 
 from ngwmn.services import ServiceException
 from ngwmn.services.ngwmn import (
-    generate_bounding_box_values, get_iddata, get_water_quality, get_well_log)
+    generate_bounding_box_values, get_iddata, get_water_quality, get_well_log, get_statistic, get_statistics)
+from .mock_data import MOCK_WELL_LOG_RESPONSE, MOCK_WQ_RESPONSE, MOCK_OVERALL_STATS, MOCK_MONTHLY_STATS
+
+
+class TestGetStatistics(TestCase):
+
+    def tearDown(self):
+        """
+            This Tear Down replaces ngwmn.get_statistics with the original.
+            Note, that his could be replaced with a with-clause mock.
+        """
```

Comment:
The linters we're using probably don't check this, but with the style guide, we indent the docstring text at the same level as the `"""`.

Reply:
fixed in next push